app/api/v1/models/election.py

```diff
@@ -46,6 +51,38 @@ class CiphertextElectionContext(Base):
     crypto_extended_base_hash: str
     """The `extended base hash code (𝑄')` in [ElectionGuard Spec](https://github.com/microsoft/electionguard/wiki)"""
 
+    @staticmethod
+    def stringToElementModP(value: str) -> ElementModP:
+        elementModP = hex_to_p(value)
+        if elementModP is None:
+            raise HTTPException(
+                status_code=status.HTTP_400_BAD_REQUEST, detail="invalid key"
+            )
+        return elementModP
+
+    @staticmethod
+    def stringToElementModQ(value: str) -> ElementModQ:
+        elementModQ = hex_to_q(value)
+        if elementModQ is None:
+            raise HTTPException(
+                status_code=status.HTTP_400_BAD_REQUEST, detail="invalid key: " + value
+            )
+        return elementModQ
+
+    def toSdkContext(self) -> CiphertextElectionContext:
```

Comment:
I would take an approach where since this depends on a base that this could use a `Protocol` and then uses a common name. `to_sdk_format` or something seems more flexible.